For various reasons sometimes the debcargo-conf patches can't be applied as-is,
so we further patch them here. The reasons are as follows:

- commoncrypto, commoncrypto-sys: these crates were released before crates.io
  automatically re-wrote all Cargo.toml files. cargo-vendor outputs the
  un-re-written original Cargo.toml files, but debcargo will forcibly rewrite
  even un-re-written ones that were published to crates.io. The patch below
  rewrites our existing patches so they work against the un-re-written ones.

- tempfile: has dependency relaxation that applies but doesn't match the vendored copy
- hashbrown, indexmap, regex-automata: outdated in debcargo-conf, would require
  big transitions at the moment, patches only fix tests/disable unpackaged
  dependencies

diff --git a/src/commoncrypto-sys/debian/patches/no-clippy.patch b/src/commoncrypto-sys/debian/patches/no-clippy.patch
index 3e4c8850e..93789f3b8 100644
--- a/src/commoncrypto-sys/debian/patches/no-clippy.patch
+++ b/src/commoncrypto-sys/debian/patches/no-clippy.patch
@@ -1,17 +1,16 @@
---- a/Cargo.toml	2017-01-24 06:56:51.000000000 +0000
-+++ b/Cargo.toml	2018-09-15 15:37:47.602333479 +0000
-@@ -19,14 +19,8 @@
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -8,13 +8,8 @@
  keywords = ["crypto", "hash", "digest", "osx", "commoncrypto"]
  license = "MIT"
- repository = "https://github.com/malept/rust-commoncrypto"
--[dependencies.clippy]
--version = "0.0"
--optional = true
  
- [dependencies.libc]
- version = "0.2"
- [dev-dependencies.hex]
- version = "0.2"
--
 -[features]
 -lint = ["clippy"]
+-
+ [dependencies]
+ libc = "0.2"
+ 
+-clippy = { version = "0.0", optional = true }
+-
+ [dev-dependencies]
+ hex = "0.2"
diff --git a/src/commoncrypto/debian/patches/no-clippy.patch b/src/commoncrypto/debian/patches/no-clippy.patch
index 38d9c9255..b21a7cae5 100644
--- a/src/commoncrypto/debian/patches/no-clippy.patch
+++ b/src/commoncrypto/debian/patches/no-clippy.patch
@@ -1,17 +1,16 @@
---- a/Cargo.toml	2017-01-24 06:56:51.000000000 +0000
-+++ b/Cargo.toml	2018-09-15 17:37:01.354423224 +0000
-@@ -19,14 +19,8 @@
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -8,13 +8,8 @@
  keywords = ["crypto", "hash", "digest", "osx", "commoncrypto"]
  license = "MIT"
- repository = "https://github.com/malept/rust-commoncrypto"
--[dependencies.clippy]
--version = "0.0"
--optional = true
  
- [dependencies.commoncrypto-sys]
- version = "0.2.0"
- [dev-dependencies.hex]
- version = "0.2"
--
 -[features]
 -lint = ["clippy"]
+-
+ [dependencies]
+ commoncrypto-sys = { version = "0.2.0", path = "../commoncrypto-sys" }
+ 
+-clippy = { version = "0.0", optional = true }
+-
+ [dev-dependencies]
+ hex = "0.2"
diff --git a/src/hashbrown/debian/patches/series b/src/hashbrown/debian/patches/series
index b84c40f49..e961ca4ee 100644
--- a/src/hashbrown/debian/patches/series
+++ b/src/hashbrown/debian/patches/series
@@ -1 +1 @@
-disable-alloc.diff
+#disable-alloc.diff
diff --git a/src/indexmap/debian/patches/series b/src/indexmap/debian/patches/series
index 8804a7d0c..4020f576f 100644
--- a/src/indexmap/debian/patches/series
+++ b/src/indexmap/debian/patches/series
@@ -1,2 +1,2 @@
-drop-rustc-rayon.diff
-drop-quickcheck.diff
+#drop-rustc-rayon.diff
+#drop-quickcheck.diff
diff --git a/src/regex-automata/debian/patches/series b/src/regex-automata/debian/patches/series
index b41d6ce94..b0c6f806a 100644
--- a/src/regex-automata/debian/patches/series
+++ b/src/regex-automata/debian/patches/series
@@ -1,3 +1,3 @@
-disable-tests-missing-testdata.patch
-remove-regex-test.patch
-relax-dep.diff
+#disable-tests-missing-testdata.patch
+#remove-regex-test.patch
+#relax-dep.diff
diff --git a/src/tempfile/debian/patches/series b/src/tempfile/debian/patches/series
index 75abb7f14..2378a84ba 100644
--- a/src/tempfile/debian/patches/series
+++ b/src/tempfile/debian/patches/series
@@ -1 +1 @@
-relax-dep.diff
+#relax-dep.diff
